features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/ndk/DatadogNdkCrashEventHandler.kt

```diff
@@ -108,6 +111,10 @@ internal class DatadogNdkCrashEventHandler(
             user?.email != null || additionalUserProperties.isNotEmpty()
         val deviceInfo = datadogContext.deviceInfo
 
+        val sourceType = sourceTypeStr?.let {
+            ErrorEvent.SourceType.fromJson(sourceTypeStr)
```

Comment:
I think this will throw if wrong value is provided, do we want to handle this case?